### DISCUSSION TITLE
[NODRIVER] how to handle with result of evaluate? I got  [{'raphael': True, 'raphaelid': 'j4ah8'}]

### DISCUSSION THREAD

**Discussion by PacharaXx** [ultrafunkamsterdam/undetected-chromedriver#2062] [Q&A]
`elements = await self.page.evaluate(script, return_by_value=True)`
 I use js for query and got this [{'raphael': True, 'raphaelid': 'j4ah8'}]
 so how can I handle with it? (click)

**Comment by nathanfallet** [comment]
You question is not clear sir, can you explain it better?

**Comment by ultrafunkamsterdam** [comment]
You are evaluating in J's. The response is pure esthetic.  How would you click a Js object you think? Tell us what you need and show whole script.
